fiasko_bro/code_validator.py

```diff
@@ -244,6 +246,9 @@ def validate(self, repo_path, original_repo_path=None, **kwargs):
         self.validator_arguments.update(kwargs)
         self.validator_arguments['whitelists'] = self.whitelists
         self.validator_arguments['blacklists'] = self.blacklists
+        max_num_of_py_files = self.validator_arguments['max_num_of_py_files']
+        if are_repos_too_large(repo_path, max_num_of_py_files, original_repo_path):
+            return [('repo is too large', '')]
```

Comment:
Maybe we should make `are_repos_too_large` singular? I mean, look at the error slug.